src/main/generated/resources/data/minestuck/minestuck/grist_cost_generation_recipes.json

```diff
@@ -19,11 +19,6 @@
     "source": "stonecutting",
     "interpreter_type": "minestuck:default"
   },
-  {
-    "source_type": "recipe_type",
-    "source": "smithing",
-    "interpreter_type": "minestuck:smithing"
-  },
```

Comment:
Can you revert this change? (Or alternatively fix it in the data provider, though that's already being done in #359)